oauth2/oauth2-resource-server/src/main/java/org/springframework/security/oauth2/server/resource/authentication/JwtAuthenticationToken.java

```diff
@@ -56,6 +70,19 @@ public JwtAuthenticationToken(Jwt jwt, Collection<? extends GrantedAuthority> au
 		this.setAuthenticated(true);
 	}
 
+	/**
+	 * Sub-class constructor.
```

Comment:
Thanks for adding a JavaDoc. Would you please have it match a bit closer to the other constructor JavaDoc?